### PR TITLE
remove ipm link

### PR DESCRIPTION
**Pull request by yichoi**
Issue #388 
Remove ipm link in http://iotjs.net/

IoT.js-DCO-1.0-Signed-off-by: Youngil Choi duddlf.choi@samsung.com

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -77,7 +77,6 @@
           <ul class="nav navbar-nav">
             <li class="active"><a href="#">Home</a></li>
             <li><a href="http://github.com/Samsung/iotjs">View on Github</a></li>
-            <li><a href="https://github.com/Samsung/iotjs/wiki/IoT.js-Package">ipm</a></li>
 			<li><a href="http://www.jerryscript.net">Powered by <b>JerryScript</b></a></li>
```
Comment (LaszloLango) [inline index.html]: A bit nitpicky, but could you fix the indentation of this line if you touch the file anyway? :)

**Comment by LaszloLango** [comment]
LGTM

**Comment by akosthekiss** [comment]
LGTM

**Comment by akosthekiss** [comment]
but, please, squash the two commits before the merge

**Comment by yichoi** [comment]
squashed and merged at [here](https://github.com/Samsung/iotjs/tree/gh-pages)

**Pull request merged**